Worksheets/worksheet_01.txt

```diff
@@ -79,67 +92,98 @@
      area = pi  * radius ** 2
      print(area)
      
+     There are several uneeded steps.
+     
  11. Explain the mistake in the following code:
      
      x = 4
      y = 5
      a = ((x) * (y))
      print(a)
      
+     The variables didnt need to be in ()
+     
  12. Explain the mistake in the following code:
      
      x = 4
      y = 5
      a = 3(x + y)
      print(a)
      
+      The 3 isnt defined and the program doesnt know what to do with the 3. the program doesnt know to apply the distrubitive preoperty.
+     
+     
  13. Explain the mistake in the following code:
      
      radius = input(float("Enter the radius:"))
      
+     There is no command to print. so it takes what you say and does the things but doesnt ever show back up.
+     
  14. Do all these print the same value? Which one is better to use and why?
      
      print(2/3+4)
      print(2 / 3 + 4)
-     print(   2 /    3+    4  )
+     print(   2 /    3+    4  )]
+     

```

Comment:
you did not actually answer the questions you were given.  There should be two answers. a Yes/No, and a description of which is best and WHY